### DISCUSSION TITLE
Multiclass model adoption

### DISCUSSION THREAD

**Discussion by dxnico29** [open-edge-platform/anomalib#2089] [Feature Requests]
Any plan to adopt multiclass models? Like the one https://github.com/zhangzjn/ADer did.

**Comment by samet-akcay** [comment]
Yes, we are currently evaluating the models to decided which one to start first.

**Reply by dxnico29** [reply]
may i know which models you are considering?

**Reply by nicozhou7** [reply]
https://lewandofskee.github.io/projects/MambaAD/ this one looks promising
